decentraland/unity-explorer
# introduce vsync toggle in settings menu

## What does this PR change?
Fix #227 


Introduce a VSync toggle in order to enable/disable the VSync

## How to test the changes?



1. Launch the explorer
2. Open the settings menu and enable the VSync
3. Your FPS should match the monitor refresh rate (and the FPS limiter dropdown should not be interactable) or a fraction of it, depending on machine's power
4. Disable the VSync
5. Your FPS should match the value you have set in the FPS limiter dropdown (which should be now interactable)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
